### ISSUE TITLE
Add support for deploying on Azure Government

### ISSUE THREAD

**Issue by jaredhocutt** [jaredhocutt/openshift4-deploy#42]

**Comment by dudash** [comment]
related to #41 - I assume this would come after.

**Comment by jaredhocutt** [comment]
> related to #41 - I assume this would come after.

Yes, that's the plan. I want to get it working in the commercial regions first and then make any modifications that are required for MAG (if any) for this RFE.
